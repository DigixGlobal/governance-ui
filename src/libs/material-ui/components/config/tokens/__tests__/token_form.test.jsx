import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import TokenForm from '../token_form';

describe('<TokenForm />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <TokenForm
        formData={{}}
        formChange={jest.fn}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
