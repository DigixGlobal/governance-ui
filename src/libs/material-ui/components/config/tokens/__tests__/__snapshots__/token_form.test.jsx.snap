// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TokenForm /> renders correctly 1`] = `
<FormField>
  <FormGroup
    widths="equal"
  >
    <FormField
      formChange={[Function]}
      formData={Object {}}
      label="Name"
      name="name"
      placeholder="e.g. \`Digix DAO\`"
    />
    <FormField
      formChange={[Function]}
      formData={Object {}}
      label="Symbol"
      name="symbol"
      placeholder="e.g. \`DGD\`"
    />
  </FormGroup>
  <FormGroup
    widths="equal"
  >
    <FormField
      formChange={[Function]}
      formData={Object {}}
      label="Address"
      name="address"
      placeholder="e.g. \`0x1234...cdef\`"
    />
    <FormField
      formChange={[Function]}
      formData={Object {}}
      label="Decimals"
      name="decimals"
      placeholder="e.g. \`3\`"
      type="number"
    />
  </FormGroup>
  <FormGroup
    widths="equal"
  >
    <FormField>
      <label
        htmlFor="metwork"
      >
        Network
      </label>
      <Connect(NetworkSelector)
        formChange={[Function]}
        formData={Object {}}
        name="network"
        showDisabled={true}
      />
    </FormField>
    <ColorPicker
      formChange={[Function]}
      formData={Object {}}
      label="Token Color"
    />
  </FormGroup>
</FormField>
`;
